packages/sunburst/package.json

```diff
@@ -35,12 +35,16 @@
         "@nivo/tooltip": "workspace:*",
         "@types/d3-hierarchy": "^1.1.8",
         "d3-hierarchy": "^1.1.8",
+        "d3-scale": "^4.0.2",
         "lodash": "^4.17.21"
     },
     "peerDependencies": {
         "react": ">= 16.14.0 < 19.0.0"
     },
     "publishConfig": {
         "access": "public"
+    },
+    "devDependencies": {
+        "@types/d3-scale": "^4.0.8"
```

Comment:
I usually put the types in `dependencies`, as for `@types/d3-hierarchy`, in general types are only needed at compile time, but in case these typings are required for the types exported by nivo, this could be an issue.